File: Session_01_InputAndOutput/go/basics.go
package main

import "fmt"

func main() {
    // 1. Output
    fmt.Println("Hello from Go!")

    // 2. Variables
    age := 19
    gpa := 3.75
    grade := 'A'
    name := "Budi"

    fmt.Println("Name:", name)
    fmt.Printf("Age: %d | GPA: %.2f | Grade: %c\n", age, gpa, grade)

    // 3. Input
    fmt.Print("Enter a new age: ")
    fmt.Scanf("%d", &age)

    fmt.Println("Updated Age:", age)
}

File: Session_01_InputAndOutput/php/basics.php
<?php
// 1. Output
echo "Hello from PHP!\n";

// 2. Variables (prefixed with $)
$age = 19;
$gpa = 3.75;
$grade = 'A';
$name = "Budi";

echo "Name: $name\n";
echo "Age: $age | GPA: $gpa | Grade: $grade\n";

// 3. Input
echo "Enter a new age: ";
$input = fgets(STDIN);
$age = (int)$input;

echo "Updated Age: $age\n";
?>
